src/visual/aria.ts

```diff
@@ -216,9 +216,10 @@ export default function ariaVisual(ecModel: GlobalModel, api: ExtensionAPI) {
 
                     const middleSeparator = labelModel.get(['data', 'separator', 'middle']);
                     const endSeparator = labelModel.get(['data', 'separator', 'end']);
+                    const columnsToExclude = labelModel.get(['data', 'columnsToExclude']);
                     const dataLabels = [];
                     for (let i = 0; i < data.count(); i++) {
-                        if (i < maxDataCnt) {
+                        if (i < maxDataCnt && !columnsToExclude?.includes(i)) {
```

Comment:
`i` is the index of data. How can we filter certain dimensions by checking index of data? The current code is more like `dataIndexToExclude` than `columnsToExclude`.

Reply:
Thanks, how about [bcd1a81](https://github.com/apache/echarts/pull/20038/commits/bcd1a81f033eb4a7ef85ee259885e0ced99adbce) instead?